roles/analysis/tasks/analysis-c2r.yml

```diff
@@ -0,0 +1,40 @@
+---
+# TODO: Install convert2rhel for different OS?
+- name: Install packages for pre-conversion analysis on CentOS 7
+  ansible.builtin.package:
+    name: "{{ analysis_packages_centos7 }}"
+    # enablerepo: "{{ analysis_repos_centos7 }}" TODO?
+    state: latest
+  when: ansible_distribution == "CentOS" and ansible_distribution_major_version|int == 7
+
+- name: Ensure convert2rhel log directory exists
```

Comment:
This might not exist at all in the beginning, so... I wonder if this check is here to actually make sure that it exists (*and it should exist*), or otherwise...

Reply:
:shrug: I added it, but I think it could be removed altogether, leapp probably has it only because they want to populate answerfile with next task - which is in same log directory

Reply:
Removed it